package.json

```
@@ -4,7 +4,7 @@
   "license": "BSD-3-Clause",
   "main": "dist/runtime.js",
   "bin": "dist/bin.js",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

```suggestion
  "version": "2.1.0",
```

version bump is done by `yarn publish` so pre-bumping doesn't work.